### PR TITLE
Move code related to the Ebook 

### PR DESCRIPTION
**Pull request by ehsansasanian**

### CODE REVIEW

**Review by dkapil — CHANGES_REQUESTED** [review]

File: persistence-modules/spring-data-jpa-simple/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This module contains sample e-book code. Therefore, it should not be used to add code for any new article.
```
[the remark this answers is not in the document]
Comment (alvin577austria) [reply persistence-modules/spring-data-jpa-simple/README.md:1]: Thanks @dkapil 
Hi @ehsansasanian, please update the README with a similar format. Thanks!

[the remark this answers is not in the document]
Comment (ehsansasanian) [reply persistence-modules/spring-data-jpa-simple/README.md]: @alvin577austria Please check https://github.com/eugenp/tutorials/pull/15297/commits/5e22e56a1e5d78926c7eb05633ef86d35d09f7fa and let me know if you need anything else.

[the remark this answers is not in the document]
Comment (alvin577austria) [reply persistence-modules/spring-data-jpa-simple/README.md]: The readme file looks ok to me.

**Comment by alvin577austria** [comment]
Hi @ehsansasanian 
I have made the initial changes to upgrade the spring-boot-2 submodule to support SpringBoot3
Can you please verify on your end if the build succeeds as well.
Next step is to combine the two submodules into one that is spring-data-jpa-simple module.
Please proceed with this step and let me know if anything is unclear.
Thanks!

**Pull request closed without merging**